code/modules/admin/topic.dm

```diff
@@ -1829,9 +1829,8 @@
 			to_chat(owner, "Standby. Reload cycle in progress. Gunnery crews ready in five seconds!")
 			return
 
-		GLOB.BSACooldown = 1
-		spawn(50)
-			GLOB.BSACooldown = 0
+		GLOB.BSACooldown = TRUE
+		addtimer(VARSET_CALLBACK(GLOB, BSACooldown, FALSE), 5 SECONDS)
```

Comment:
This kinda feels like an unrelated change for the PR

Reply:
It kinda is, but while I was at the source of these vars I had to do something about this